src/panels/lovelace/cards/hui-humidifier-card.ts

```diff
@@ -112,24 +119,46 @@ export class HuiHumidifierCard extends LitElement implements LovelaceCard {
           ></round-slider>
         `;
 
-    const setValues = html`
-      <svg viewBox="0 0 24 20">
-        <text x="50%" dx="1" y="73%" text-anchor="middle" id="set-values">
-          ${isUnavailableState(stateObj.state) ||
-          setHumidity === undefined ||
-          setHumidity === null
+    const mainHumidity = html`
+      <svg viewBox="0 0 30 20">
+        <text x="50%" dx="1" y="73%" text-anchor="middle" id="main-humidity">
+          ${curHumidity
+            ? svg`
+                      ${curHumidity}
+                      <tspan dx="-3" dy="-6.5" style="font-size: 4px;">
+                        %
+                      </tspan>
+                      `
+            : isUnavailableState(stateObj.state) ||
+              setHumidity === undefined ||
+              setHumidity === null
             ? ""
             : svg`
-                    ${setHumidity.toFixed()}
-                    <tspan dx="-3" dy="-6.5" style="font-size: 4px;">
-                      %
-                    </tspan>
-                    `}
+                        ${setHumidity.toFixed()}
```

Comment:
Use `formatNumber` (I know this was not used before, but it should)

Reply:
Ok